__tests__/e2e/synthetics.journey.ts

```diff
@@ -0,0 +1,195 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020-present, Elastic NV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { beforeAll, journey, step, expect } from '@elastic/synthetics';
+import axios from 'axios';
+
+beforeAll(async () => {
+  await waitForElasticSearch();
+});
+
+async function logIn(page) {
+  await page.fill('[data-test-subj="loginUsername"]', 'elastic');
+  await page.fill('[data-test-subj="loginPassword"]', 'changeme');
+  await page.click('[data-test-subj="loginSubmit"]');
+}
+
+async function goToSyntheticsIntegrationPage(page) {
+  console.info('Navigating to Elastic Synthetics Integration page')
+  await page.goto('http://localhost:5601/app/integrations/detail/synthetics/overview');
+  await page.waitForSelector('[data-test-subj="loginUsername"]', { timeout: 10000 });
+  const isUnauthenticated = await page.isVisible('[data-test-subj="loginUsername"]');
+  if (isUnauthenticated) {
+    await logIn(page);
+  }
+  await page.click('[data-test-subj="addIntegrationPolicyButton"]', { timeout: 10 * 1000 });
+  /* We need to ensure that Elastic Synthetics integration page is fully loaded, including the UI logic.
+   * Our UI logic clears out the name input when the page is first loaded. If we don't wait, we run the risk 
+   * of playwright input being overwritten by Kibana logic clearing out the field. Playwright doesn't have
+   * a mechanism to wait for the value of input to be empty, so for now, we are using a simple timeout */
+  await page.waitForTimeout(10 * 1000);
+}
+
+async function goToUptime(page) {
+  console.info('Navigating to Uptime overview page')
+  await page.goto('http://localhost:5601/app/uptime');
+  await page.waitForSelector('h1', { timeout: 10000 });
```

Comment:
cant we search for something better here? Like `text=Monitors` or some headlines with explicit title?

Reply:
The h1 is actually a different between different versions, due to the recent "No data" screen. So this accounts for all of the variations. I could also do `await page.waitForUrl("")` instead if you prefer.

Reply:
Hmmm gotcha, if its going to be a explicit URL then it would be better. I have found it hard when trying to debug these tests before. So more verbose would be always preferable.